### PR TITLE
docs: Add link before YouTube videos to retrigger the CookieConsent scripts

### PR DESCRIPTION
**Pull request by JohannesDienst-askui**
So this seems to be the only viable solution to get YouTube videos working with Cookie Consent bot.

The problem is that the CookieConsent Bot is trigger **exactly once** at page load.

Docusaurus is a SPA and does no page reload when you navigate to another side.

But the iframe you want to load when marketing cookies are enabled gets loaded only when you navigate to it (which is not a page refresh).

Thus the Cookie Consent scripts never "see" it and therefore it is disabled.

Solution
---------------
Added a link that triggers a rerun of the Cookie Consent scripts.

Not working Alternatives
---------------
* Add onload event to iframes -> Cookie Consent messes with the HTML of the iframe and removes the onload attribute

### CODE REVIEW

File: docs/docs/general/06-Tutorials/zip-images-upload-googledrive-windows.md [hunk 1]
```diff
@@ -9,8 +9,11 @@ This tutorial will show you how to zip files on your file system and then upload
 <div class="cookieconsent-optout-marketing">
   Please <a href="javascript:Cookiebot.renew()">accept marketing-cookies</a> to watch this video.
 </div>
+<div class="cookieconsent-optin-marketing">
+  <a href="javascript:rerunCookieConsentScripts(0)">Please click to watch this video.</a>
+</div>
+<iframe onload="rerunCookieConsentScripts(0);" width="560" height="315" data-cookieblock-src="https://www.youtube-nocookie.com/embed/i3M5SPYQTKI&embedded=true" data-cookieconsent="marketing" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (adi-wan-askui) [inline docs/docs/general/06-Tutorials/zip-images-upload-googledrive-windows.md:15]: Didn't you say that the `onload` does not work? 😄

File: docs/docs/general/06-Tutorials/zip-images-upload-googledrive-windows.md [hunk 2]
```diff
@@ -9,8 +9,11 @@ This tutorial will show you how to zip files on your file system and then upload
 <div class="cookieconsent-optout-marketing">
   Please <a href="javascript:Cookiebot.renew()">accept marketing-cookies</a> to watch this video.
 </div>
+<div class="cookieconsent-optin-marketing">
+  <a href="javascript:rerunCookieConsentScripts(0)">Please click to watch this video.</a>
```
Comment (adi-wan-askui) [inline docs/docs/general/06-Tutorials/zip-images-upload-googledrive-windows.md:13]: Can you remove the retry parameter? (You can still do a retry but the consumer does not really need to care.)

**Review by adi-wan-askui — APPROVED** [review]
Well figured out. Nice and simple solution ;) 

I wonder if it is also possible to listen to or intercept the routing events -- I guess that they are using react-router -- and run the script after having navigated there.

File: docs/docs/general/06-Tutorials/zip-images-upload-googledrive-windows.md (hunk 1 above)
Comment (JohannesDienst-askui) [reply docs/docs/general/06-Tutorials/zip-images-upload-googledrive-windows.md:15]: Will be removed 😉

**Pull request merged**